### PR TITLE
91_version_history file updates for 2.1.1 release

### PR DESCRIPTION
**Pull request by Mohit04tomar**

### CODE REVIEW

**Review by praveen-garg — COMMENTED** [review]
1) We should group the related items 
2) list plugins released as packages first time vs fixed
3) keep documentation update/fixes at the bottom of each section; and major highlights on top

File: docs/91_version_history.rst [hunk 1]
```diff
@@ -25,6 +25,71 @@ Version History
...
+- **Services & Plugins**
+
+    - New Features:
+
+       - A new south plugin, fledge-south-J1939, has been created to collect data into Fledge.
+       - Support for multiple data centers for OSIsoft Cloud Services (OCS) has been added. OCS is hosted in the US-West and EU-West regions.
+       - There have been improvements to the OMF north plugin to prevent an issue that could cause the plugging to stop sending data if the type of an individual datapoint changed repeatedly between integer and floating point values. The logging of the the plugin has also been improved, with clearing messages and less repetition of error conditions that persist for long periods.
+       - The audit logger has been made available to plugins running within the notification service.
+       - The control dispatcher now has access to the audit logging system.
+       - The notification service documentation has been updated to include examples of notifications based on statistics and audit logs.
```
Comment (ashish-jabble) [inline docs/91_version_history.rst:75]: We may move this down to the end of the list.

File: docs/91_version_history.rst [hunk 2]
```diff
@@ -25,6 +25,71 @@ Version History
...
+       - The notification mechanism has been updated to accept raw statistics and statistics rates as an input for notification rules. This allows alerts to be raised for pipeline flows and other internal tasks that generate statistics.       
+       - Notifications can now register for audit log entries to be sent to notification rules. This allows notification to be made based on internal state changes of the system.
+       - A number of optimisations to the SQLite in-memory storage plugin and the SQLiteLB storage plugin have been added that increase the rate at which readings can be stored with these plugins.
+       
+
+    - Bug Fix:
+       
+       - An issue with the Modbus-TCP & S7 plugins which caused the polling to fail has been resolved.
+       - The control map configuration item of the Modbus C plugin was incorrectly described, this has now been resolved.
+       - A problem with the J1708 & J1939 plugins that caused them to fail if added disabled and then later enabling them has been resolved.
```
Comment (ashish-jabble) [inline docs/91_version_history.rst:85]: J1939 is the new addition so remove it from here.

File: docs/91_version_history.rst [hunk 3]
```diff
@@ -25,6 +25,71 @@ Version History
...
+       - Changes have been made to the purge process in the sqlitelb and sqliteinmemory plugins in order to improve performance.
+       - A change has been made to the configuration of the storage plugin such that rather than having to type correct names for storage plugins the user may now select the plugins to use from a drop down list. Note however that the system must still be restarted for the new storage plugin to take effect.
+       - The storage service has been updated to allow other services to subscribe the notifications of inserts into the generic tables.
+       - The documentation regarding handling and updating certificates used for authentication has been updated.
+
+
+    - Bug Fix:
+
+       - An issue that caused the retrieval of system logs for services with white space in the name of the service has been resolved.
+       - An erroneous message was being produced when starting the system using the SQLite in memory storage plugin. This has now been resolved.
```
Comment (praveen-garg) [inline docs/91_version_history.rst:51]: SQLite in-memory

File: docs/91_version_history.rst [hunk 4]
```diff
@@ -25,6 +25,71 @@ Version History
...
+- **Fledge Core**
+
+    - New Features:
+       
+       - The south plugin now supports three different modes of polling. Polling at fixed intervals from the time started, polling at fixed times or polling on demand via the control mechanisms.
+       - Plugins that run within the south and north services and north tasks now have access to the audit logging system.
+       - A number of optimisations have been made to improve the performance of Python filters within a pipeline.
+       - The support bundle has been updated to include a list of the Python packages installed on the machine.
+       - The configuration category C++ API has been enhanced in the retrieval and setting of all the attributes of a configuration item.
+       - Changes have been made to the purge process in the sqlitelb and sqliteinmemory plugins in order to improve performance.
```
Comment (praveen-garg) [inline docs/91_version_history.rst:42]: SQLIteLB and SQLite In-memory

File: docs/91_version_history.rst [hunk 5]
```diff
@@ -25,6 +25,71 @@ Version History
...
+       - A change has been made to the configuration of the storage plugin such that rather than having to type correct names for storage plugins the user may now select the plugins to use from a drop down list. Note however that the system must still be restarted for the new storage plugin to take effect.
```
Comment (praveen-garg) [inline docs/91_version_history.rst:43]: drop-down

File: docs/91_version_history.rst [hunk 6]
```diff
@@ -25,6 +25,71 @@ Version History
...
+       - The storage service has been updated to allow other services to subscribe the notifications of inserts into the generic tables.
+       - The documentation regarding handling and updating certificates used for authentication has been updated.
+
+
+    - Bug Fix:
+
+       - An issue that caused the retrieval of system logs for services with white space in the name of the service has been resolved.
+       - An erroneous message was being produced when starting the system using the SQLite in memory storage plugin. This has now been resolved.
+       - Support has been improved for switching between different storage plugins that allows for correct schema creation when using different sqlite plugin variants for configuration and readings storage.
```
Comment (praveen-garg) [inline docs/91_version_history.rst:52]: SQLite

File: docs/91_version_history.rst [hunk 7]
```diff
@@ -25,6 +25,71 @@ Version History
...
+       - A text wrapping issue in the system log viewer has been resolved.
+
+
+- **Services & Plugins**
+
+    - New Features:
+
+       - A new south plugin, fledge-south-J1939, has been created to collect data into Fledge.
+       - Support for multiple data centers for OSIsoft Cloud Services (OCS) has been added. OCS is hosted in the US-West and EU-West regions.
+       - There have been improvements to the OMF north plugin to prevent an issue that could cause the plugging to stop sending data if the type of an individual datapoint changed repeatedly between integer and floating point values. The logging of the the plugin has also been improved, with clearing messages and less repetition of error conditions that persist for long periods.
```
Comment (praveen-garg) [inline docs/91_version_history.rst:72]: plugging?

Comment (praveen-garg) [reply docs/91_version_history.rst:72]: The logging of the the plugin -> The logging of the plugin

BTW plugin word used too many times

File: docs/91_version_history.rst [hunk 8]
```diff
@@ -25,6 +25,71 @@ Version History
...
+       - The notification service documentation has been updated to include examples of notifications based on statistics and audit logs.
+       
+
+    - Bug Fix:
+       
+       - An issue with the Modbus-TCP & S7 plugins which caused the polling to fail has been resolved.
+       - The control map configuration item of the Modbus C plugin was incorrectly described, this has now been resolved.
+       - A problem with the J1708 plugin that caused them to fail if added disabled and then later enabling them has been resolved.
+       - A problem that caused the Azure IoT Core north plugin to fail to send data has been corrected.
+       - The plugin would become unresponsive if the OPC UA server was unavailable or if the server URL was incorrect. The only way to stop the plugin in this state was to shut down Fledge. This has been fixed.
```
Comment (praveen-garg) [inline docs/91_version_history.rst:87]: which plugin?

**Review by MarkRiddoch — APPROVED** [review]

**Pull request closed without merging**